### ISSUE TITLE
[Bug] White spaces are incompatible with box mode

### ISSUE THREAD

**Issue by T0nyX1ang** [T0nyX1ang/noqx#62]
One can deal with white spaces with box mode, without changing the white spaces in the grid settings.

A possible solution is to add detections from the valid points:
```py
        import json
        from itertools import accumulate
        
        points = list(accumulate(json.loads(self.parts[5])))
        for point in points:
            print(self.index_to_coord(point)[0])
```

**Comment by T0nyX1ang** [comment]
A workaround is to allow users to input the white spaces without re-creating a new puzzle, and this is approachable now.

**Comment by T0nyX1ang** [comment]
Fixed in commit 249834d.
